build.gradle

```diff
@@ -67,12 +67,13 @@ ext {
     moduleName = "io.aiven.kafka.connect.http"
 
     avroVersion = "1.8.1" // Version 1.8.1 brings Jackson 1.9.x/org.codehaus.jackson package for Avro and Confluent Platform 4.1.4.
-    confluentPlatformVersion = "4.1.4" // For compatibility tests use version 4.1.4.
+    confluentPlatformVersion = "6.0.2" // For compatibility tests use version 6.0.2 to match Kafka 2.6.
```

Comment:
I'm not sure we can use this version as they might contain an incompatible license.

cc. @willyborankin

Reply:
@jlprat we are not using Confluent Platform itself here, what we use is just
```
integrationTestRuntimeOnly "io.confluent:kafka-avro-serializer:$confluentPlatformVersion"
integrationTestRuntimeOnly "io.confluent:kafka-connect-avro-converter:$confluentPlatformVersion"
integrationTestRuntimeOnly "io.confluent:kafka-json-serializer:$confluentPlatformVersion"
```
and all of these libraries have Apache 2.0:
https://github.com/confluentinc/schema-registry/blob/master/avro-serializer/LICENSE
https://github.com/confluentinc/schema-registry/blob/master/json-serializer/LICENSE
https://github.com/confluentinc/schema-registry/blob/master/avro-converter/LICENSE

Perhaps the name of the variable is a bit misleading.

Reply:
I think the name is fine, because we pull the Confluent Platform binary, and only use the right libraries in there. But this is an area we need to be careful. Just because some libraries might be at one's disposal it doesn't mean they can be used. This is more for subsequent PRs. If I'm not mistaken under Confluent Platform 4.1.4 all components were Apache Licensed v2, after this change only a subset of them are (like the ones you mention)

Reply:
Exactly, that's why the first thing I did when I started the review is checked the licenses.